go.mod

```diff
@@ -38,7 +38,7 @@ require (
 	github.com/getlantern/eventual v1.0.0
 	github.com/getlantern/eventual/v2 v2.0.2
 	github.com/getlantern/filepersist v0.0.0-20210901195658-ed29a1cb0b7c
-	github.com/getlantern/flashlight/v7 v7.6.111
+	github.com/getlantern/flashlight/v7 v7.6.112-0.20240929144745-72a54c5a4e73
```

Comment:
This includes changes from the unmerged flashlight counterpart: https://github.com/getlantern/flashlight/pull/1416

When I've tested these changes thoroughly, I will merge the flashlight PR in and update this reference.